src/core/flags/new/template.flags.ts

```diff
@@ -0,0 +1,27 @@
+import { Flags } from '@oclif/core';
+
+export const templateFlags = () => {
+  return {
```

Comment:
I suggest to add also `renderer` flag as some people may use `nunjucks`, see: https://www.asyncapi.com/docs/tools/generator/template-development

Reply:
Done @Amzani